integrations/tensorflow/e2e/keras/BUILD

```diff
@@ -180,18 +187,68 @@ iree_vision_test_suite(
 )
 
 iree_vision_test_suite(
-    name = "vision_external_tests",
+    name = "long_cifar10_internal_tests",
+    size = "enormous",
```

Comment:
"long" also has a specific meaning in Bazel, so having a suite called "long" that aren't given a "long" timeout seems weird. Do these tests really need to be "enormous"? I assume the issue is they take longer than 15 minutes. Do they need more than 300MB of RAM?

If these should really be "enormous", I think the test suite name should match

```suggestion
    name = "enormous_cifar10_internal_tests",
    size = "enormous",
```

Reply:
Some targets take longer than 15 minutes at the moment :\